shared/common-adapters/markdown/react.tsx

```diff
@@ -351,20 +347,15 @@ const previewOutput = SimpleMarkdown.reactFor(
       case 'newline':
         return ' '
       case 'blockQuote':
-        return (
-          <>
-            {output([{content: '> ', type: 'text'}], state)}
-            {output(ast.content, state)}
-          </>
-        )
+        return React.Children.toArray([
```

Comment:
turns out react will complain if both outputs are strings so you have an array of strings w/o a key. output returns a raw string (even if you pass `key` to `state`. so instead use `React.children` to fix this up